### ISSUE TITLE
Recurrent BatchNorm for SRU?

### ISSUE THREAD

**Issue by cbasavaraj** [taolei87/sru#20]
Hi @taolei87, do you think your SRU could improve further with the use of BatchNorm as in [Cooijmans et al. (2017)](https://arxiv.org/abs/1603.09025), which has been implemented by @jihunchoi [here](https://github.com/jihunchoi/recurrent-batch-normalization-pytorch) for an LSTM?
Thank you.

**Comment by taolei87** [comment]
hi @cbasavaraj , this sounds very interesting. 
i will put it in the TODO, but i wouldn't be available to try this recent. Got quite many other tasks to do.
